### PR TITLE
Override hosting instructions with FTW specific

### PR DESCRIPTION
**Pull request by rap1ds**
Hosting FTW with static server doesn't seem to work anymore. This is
probably due to code splitting.

Override the default instructions with FTW specific instructions.

I tested this with Verdaccion. Seems to work fine.

Before:

Screen Shot 2021-08-03 at 14 46 59

After:

Screen Shot 2021-08-03 at 15 59 49

### CODE REVIEW

File: packages/react-scripts/scripts/build.js [hunk 1]
```diff
@@ -40,7 +40,36 @@ const configFactory = require('../config/webpack.config');
...
+  publicPath,
+  buildFolder,
+  useYarn
+) {
+  console.log();
+  console.log(`The ${chalk.cyan(buildFolder)} folder is ready to be deployed.`);
+  console.log();
+  console.log('You may serve it by running:');
+  console.log();
+  console.log(`  ${chalk.cyan('yarn')} start`);
```
Comment (Gnito) [inline packages/react-scripts/scripts/build.js:75]: Maybe "yarn run dev-server" instead?

"yarn start" is used by host service which should have NODE_ENV & PORT set in environment variables. (PORT is handled by host services pretty independently)

(So, this should work too: `NODE_ENV=development PORT=3000 yarn start`)

File: packages/react-scripts/scripts/build.js [hunk 2]
```diff
@@ -40,7 +40,36 @@ const configFactory = require('../config/webpack.config');
 const paths = require('../config/paths');
 const checkRequiredFiles = require('react-dev-utils/checkRequiredFiles');
 const formatWebpackMessages = require('react-dev-utils/formatWebpackMessages');
-const printHostingInstructions = require('react-dev-utils/printHostingInstructions');
+
+//
+// Override the default hosting instructions with FTW specific instructions.
```
Comment (Gnito) [inline packages/react-scripts/scripts/build.js]:
```suggestion
// Sharetribe custom: override the default hosting instructions with FTW specific instructions.
```

We have used this string ("Sharetribe custom:") in all the places that has been customized from default CRA project.

File: packages/react-scripts/scripts/build.js (hunk 1 above)
Comment (rap1ds) [reply packages/react-scripts/scripts/build.js:75]: @Gnito I copied the "yarn start" from this article (https://www.sharetribe.com/docs/ftw-hosting/how-to-deploy-ftw-to-production/#building-the-app) which is also linked below.

What I'm thinking is that isn't the "yarn build" command meant to build the production bundle which is meant to be run in a production mode mainly? What would be the reason to run the build bundle in dev mode?

So I'm wondering that one option would be to change the instructions from "You may serve it by running:" to "You may serve it on production mode by running:", and then the command would be `NODE_ENV=production PORT=3000 yarn start`. Would that make sense?

Comment (Gnito) [reply packages/react-scripts/scripts/build.js:75]: Yes, that makes sense. My main point is that those 2 environment variables need to be handled in some way.

NODE_ENV is used to [take proper dotenv files into use](https://github.com/sharetribe/create-react-app/blob/master/packages/react-scripts/config/env.js#L21).
PORT is something that host services usually want to handle on their own (3000 is fine for local machine)

**Pull request merged**